src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -185,7 +186,7 @@ export abstract class BaseEslintEngine extends AbstractRuleEngine {
 	async run(ruleGroups: RuleGroup[], rules: Rule[], targets: RuleTarget[], engineOptions: Map<string, string>): Promise<RuleResult[]> {
 
 		// Get sublist of rules supported by the engine
-		const configuredRules = this.configureRules(rules);
+		const configuredRules = this.configureRules(rules).rules;
```

Comment:
The method has been altered to return the redefined `ESRuleConfig` type, and we immediately dereference its `rules` property. That lets us get around the need to import `Linter.RulesRecord` or define an alias for it.